vitrina/helpers.py

```diff
@@ -449,41 +458,137 @@ def get_current_domain(request: WSGIRequest, ensure_secure=False) -> str:
             created=datetime.datetime.now(),
             version=0,
             identifier=email_identifier,
-            template=base_template_content,
+            template=email_template_to_save,
             subject=_(email_title_subject),
             title=_(email_title)
         )
         created_template.save()
     else:
         email_template = email_template.first()
         email_content = str(email_template.template)
-        email_content = email_content.format(*email_template_keys)
-        email_subject = str(email_template.subject)
+        if email_template_keys:
+            for key in list_keys:
+                if key in email_template_keys.keys():
+                    if email_template_keys[key] is not None:
+                        email_content = email_content.replace("{" + key + "}", email_template_keys[key])
+                else:
+                    email_content = email_content.replace("{" + key + "}", '')
+            email_subject = str(email_template.subject)
+
 
-    return {'email_content': email_content, 'email_subject': email_subject}
+def _get_email_tempate_from_db(name: str) -> EmailTemplate | None:
+    try:
+        return EmailTemplate.objects.get(identifier=name)
+    except EmailTemplate.DoesNotExist:
+        return None
+
+
+def email(
+    recipients: list[str],
+    email_identifier: str,
+    name: str,  # template name
+    context: dict[str, Any] | None = None,
+    *,
+    # Allow user to override email templates via Admin.
+    override: bool = True,
+) -> dict[str, str]:
+    context = context or {}
+    data_dict = {}
+    email_template = None
+    email_send = False
+    subject = None
+    if override:
+        email_template = _get_email_tempate_from_db(email_identifier)
+    if email_template:
+        from unidecode import unidecode
+        subject = email_template.subject
+        template_db = email_template.template
+        template_db = unidecode(template_db)
+        template = Template(template_db)
+        for key in template.nodelist:
+            if type(key).__name__ == 'VariableNode':
+                data_dict.update({key.token.contents: context[key.token.contents]})
+        context = Context(data_dict)
+        content = template.render(context)
+        html_message = markdown.markdown(content)
+    else:
+        template_path = get_template(name)
+        with open(template_path.origin.name, encoding="utf-8") as file:
+            read_data = file.readlines()
+        content = render_to_string(name, context)
+        html_message = markdown.markdown(content)
+        send_email_title = content.split('\n')
+        content_to_save = markdown.markdown(''.join(read_data))
+        EmailTemplate.objects.create(
+            created=datetime.datetime.now(),
+            version=0,
+            identifier=email_identifier,
+            template=content_to_save,
+            subject=read_data[0].split('\n')[0],
+            title=read_data[0].split('\n')[0]
```

Comment:
You can't split template text like that, because rendered version, must have subject on first line, but the template, might have template tags on first line.

I suggest, to store whole template, including title into single field.

Reply:
Well in my eyes, the idea was that the first line of the template must be email subject, right? So, the way I see is that subject has some tags, for example
<h6>Pakeistas duomenų rinkinys: {{title}}</h6>
and whole is saved into the database, sorry what is the problem? 

Or do you mean, title can be something like this <h6></h6> ? I mean only html tags without any real text, but in that case what`s the point to have something like that? For example if you want email without subject, you can just leave empty first line.